misago/legal/models.py

```diff
@@ -1 +1,113 @@
-# Empty models file that triggers migrations for this app
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.template.defaultfilters import slugify
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from misago.conf import settings
+
+
+class NoActiveLegalAgreement(ValidationError):
+    pass
+
+
+class LegalAgreementManager(models.Manager):
+    def get_current(self):
+        try:
+            return self.filter(is_active=True)
+        except self.model.DoesNotExist:
+            raise NoActiveLegalAgreement(
+                u'Please create an active LegalAgreement'
+            )
+
+
+class LegalAgreement(models.Model):
+    created_at = models.DateTimeField(default=timezone.now)
+    modified_at = models.DateTimeField(auto_now=True)
+    published_at = models.DateTimeField(blank=True, null=True)
+    key = models.SlugField(unique=True, blank=True)
+
+    TOS = 'terms_of_service'
+    PRIVACY = 'privacy_policy'
+    LEGAL_CHOICES = (
+        (TOS, 'Terms of service'),
+        (PRIVACY, 'Privacy Policy'),
+    )
+    title = models.CharField(
```

Comment:
Title should be user-editable field that allows any input. You should have one more `type` field where user would toggle between `TOS` and `PRIVACY`.